autospotting.go

```diff
@@ -3,12 +3,11 @@ package main
 import (
 	"encoding/json"
 	"flag"
-	"fmt"
 	"log"
 	"os"
 
-	autospotting "github.com/cristim/autospotting/core"
 	lambda "github.com/eawsy/aws-lambda-go/service/lambda/runtime"
+	autospotting "github.com/cristim/autospotting/core"
```

Comment:
I don't think it's go fmt, but it may be my use of goimports.

I will investigate this further.